### DISCUSSION TITLE
How does windows use mediamtx?

### DISCUSSION THREAD

**Discussion by 1604963180** [bluenviron/mediamtx#2833] [Questions]
### Question

How to make the rtsp video stream address play in the video tag with mediamtx?

**Comment by aler9** [comment]
Hello, the README describes how to [ingest RTSP streams](https://github.com/bluenviron/mediamtx#rtsp-cameras-and-servers) and out to [read streams from browsers](https://github.com/bluenviron/mediamtx#web-browsers-1). Conversion is automatic.

There's no way to use directly the \<video> tag, but you can use\<iframe> as mentioned in the README.

[a comment by an automated account is not shown]
